src/docs/guides/fastify.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Deploy a Fastify App
+---
+
+Fastify is a high-performance, low-overhead web framework for Node.js, designed to deliver an exceptional developer experience.
+
+This guide covers how to deploy a Fastify app on Railway in four ways:
+
+1. [One-click deploy from a template](#one-click-deploy-from-a-template).
+2. [From a GitHub repository](#deploy-from-a-github-repo).
+3. [Using the CLI](#deploy-from-the-cli).
+4. [Using a Dockerfile](#use-a-dockerfile).
+
+## One-Click Deploy from a Template
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/ZZ50Bj)
+
+You can also choose from a <a href="https://railway.app/templates?q=fastify" target="_blank">variety of Fastify app templates</a> created by the community.
+
+## Deploy from a GitHub Repo
+
+To deploy a Fastify app on Railway directly from GitHub, follow the steps below:
+
+1. Fork the basic <a href="https://github.com/railwayapp-templates/fastify-hello-world" target="_blank">fastify-hello-world GitHub repo</a>.
```

Comment:
Just curious about why not ask them to fork the repo that the fastify template deploys from?

Reply:
I wanted to showcase a Hello World API according to what's specified in my task on Linear. That's why I took this approach from scratch.

Reply:
Might be best to update the existing repo instead of creating a new one, but that's just my opinion.

Reply:
Great idea!